README.md

```diff
@@ -40,18 +55,6 @@ Output example
 
 ![Example output](https://user-images.githubusercontent.com/3704611/53132247-845f7080-356f-11e9-9c76-a9498d4a744b.png)
 
-Why even care about unsafe Rust usage?
---------------------------------------
-
-When and why to use unsafe Rust is out of scope for this project; it is simply
-a tool that provides information to aid auditing and hopefully to guide
-dependency selection. It is, however, the opinion of the author of this project
-that __libraries choosing to abstain from unsafe Rust usage when possible should
-be promoted__.
```

Comment:
> It is, however, the opinion of the author of this project that __libraries choosing to abstain from unsafe Rust usage when possible should be promoted__.

I understand dropping altogether this statement of opinion, as a resolution of #90, to be the core point of this patch.  Insofar as my opinion is relevant, I have no objection to the patch overall, but I await the opinions of tarcieri and Shnatsel on whether this statement should be dropped entirely.

Reply:
It's not dropped, it's moved.